ovector.c

```diff
@@ -17,12 +18,22 @@ void initVector(OVector* vector) {
     vector->array = createArray(vector->capacity);
 }
 
+void isOvectorNull(OVector* vector) {
```

Comment:
it's better to return `bool` value and don't print anything inside your library

This is the users' responsibility to decide what to do in such cases

And add this method to users' API